src/app/js/formats/streamgraph/Streamgraph.js

```diff
@@ -101,7 +101,8 @@ class Streamgraph extends PureComponent {
     initTheGraphBasicsElements(width, height, margin, svgViewport) {
         const zoom = d3
             .zoom()
-            .scaleExtent([1, 32])
+            //.scaleExtent([1, 32])
+            .scaleExtent([1, 572])
```

Comment:
Please resolve conflict with #874 before merging

Reply:
that's done :)